packages/gatsby-theme/src/components/Grid/GridColumn.js

```diff
@@ -26,19 +26,13 @@ const GridColumn = ({ columns, rows, children, ...props }) => {
 
 GridColumn.propTypes = {
   children: PropTypes.node.isRequired,
-  columns: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
-    PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  ]),
-  rows: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
-    PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  ]),
+  columns: PropTypes.node,
+  rows: PropTypes.node,
```

Comment:
We should be careful with `PropTypes.node`. In this case, having a type node here would mean we could pass something like `columns={<div>hello</div}`.